baybe/constraints/continuous.py

```diff
@@ -52,6 +52,17 @@ class ContinuousLinearConstraint(ContinuousConstraint):
     rhs: float = field(default=0.0, converter=float, validator=finite_float)
     """Right-hand side value of the in-/equality."""
 
+    is_interpoint: bool = field(
+        alias="interpoint", default=False, validator=instance_of(bool)
+    )
+    """Flag for defining an interpoint constraint.
+
+    While intrapoint constraints impose conditions on each individual point of a batch,
+    interpoint constraints do so **across** the points of the batch. That is, an
+    interpoint constraint of the form ``x + y <= 1`` technically encodes
+    ``sum(x_i) + sum(y_i) <= 1`` for all points ``i`` in the batch.
```

Comment:
I know what you are trying to say here, but the it's written is really confusing. Because what you write is `it encodes "some inequality"` and this encoding is done `for all points i` --> it sort of suggests that there are as many inequalities as there are points `i`. 

I'd rather rephrase it to something like "encodes `sum_i(x_i) + sum_i(y_i) <= 1`, where x and y are ..." or similar. You know what I mean?

Reply:
to clarify: if I take your statement word by word and "expand it", I'd get
* `sum(x_1) + sum(y_1) <= 1`
* `sum(x_2) + sum(y_2) <= 1`
* ...

Since you write that this constraint holds `for all i`